corehq/apps/app_manager/app_translations/generators.py

```diff
@@ -96,7 +96,10 @@ def occurrence(_row):
                 list_or_detail_index = self._get_header_index(sheet_name, 'list_or_detail')
 
                 def occurrence(_row):
-                    return ':'.join([_row[case_property_index], _row[list_or_detail_index]])
+                    case_property = _row[case_property_index]
+                    # limit case property length to avoid errors at Transifex where there is a limit of 1000
+                    case_property = case_property[:950]
+                    return ':'.join([case_property, _row[list_or_detail_index]])
```

Comment:
Can you share an example @esoergel . Sorry i didn't understand that. Are you referring to the length limit i added? If yes, that is to address very long case property calculations, i don't think anything else would be that long enough.

Reply:
Yeah, I was thinking of the case_property truncation.  What do you mean "case property calculations"?  I was just thinking if there _are_ user-defined property names that are 1000 characters long, it's conceivable that the first 950 characters of two properties might be the same.  

Users frequently use patterns like this in their property names:
`patient_2_demographic_information_age`
`patient_2_demographic_information_sex`
But at any rate, it does seem pretty unlikely that this would happen at that length.

Reply:
we found this long if expressions in excel sheet that was more than 1000 characters. Didn't come across a property name that was that long.. definitely highly unlikely someone would keep a name that long.. also looks like we wont rely on that to be unique. Might get more insight when i begin working on the second phase.